apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerUtils.java

```diff
@@ -3331,63 +3328,24 @@ public static Bundle createBundle(
    *     Patient}s, which may contain multiple matching resources, or may also be empty.
    */
   public static Bundle addResourcesToBundle(Bundle bundle, List<IBaseResource> resources) {
-    Set<String> beneIds = new HashSet<String>();
     for (IBaseResource res : resources) {
       BundleEntryComponent entry = bundle.addEntry();
       entry.setResource((Resource) res);
-
-      if (entry.getResource().getResourceType() == ResourceType.ExplanationOfBenefit) {
-        ExplanationOfBenefit eob = ((ExplanationOfBenefit) entry.getResource());
-        if (eob != null
-            && eob.getPatient() != null
-            && !Strings.isNullOrEmpty(eob.getPatient().getReference())) {
-          String reference = eob.getPatient().getReference().replace("Patient/", "");
-          if (!Strings.isNullOrEmpty(reference)) {
-            beneIds.add(reference);
-          }
-        }
-      } else if (entry.getResource().getResourceType() == ResourceType.Patient) {
-        Patient patient = ((Patient) entry.getResource());
-        if (patient != null && !Strings.isNullOrEmpty(patient.getId())) {
-          beneIds.add(patient.getId());
-        }
-      } else if (entry.getResource().getResourceType() == ResourceType.Coverage) {
-        Coverage coverage = ((Coverage) entry.getResource());
-        if (coverage != null
-            && coverage.getBeneficiary() != null
-            && !Strings.isNullOrEmpty(coverage.getBeneficiary().getReference())) {
-          String reference = coverage.getBeneficiary().getReference().replace("Patient/", "");
-          if (!Strings.isNullOrEmpty(reference)) {
-            beneIds.add(reference);
-          }
-        }
-      }
     }
-
-    logBeneIdToMdc(beneIds);
-
     return bundle;
   }
 
   /**
-   * Output list of benefificiary IDs to MDC logging
-   *
-   * @param beneIds the {@link Collection} of beneficiary IDs top log
-   */
-  public static void logBeneIdToMdc(Collection<String> beneIds) {
-    if (!beneIds.isEmpty()) {
-      BfdMDC.put("bene_id", String.join(", ", beneIds));
-    }
-  }
-
-  /**
-   * Output beneficiary ID to the MDC logging
+   * Parses Sting beneficiary ID to Long and adds to List to log to MDC.
    *
-   * @param beneId the {@link Long} beneficiary ID to log
+   * @param beneIds the collection of beneIds logged to MDC.
+   * @param beneId beneficiaryId to add to beneId collection.
    */
-  public static void logBeneIdToMdc(Long beneId) {
-    if (beneId != null) {
-      BfdMDC.put("bene_id", String.valueOf(beneId));
+  private static void addBeneIdsToSet(Set<Long> beneIds, String beneId) {
+    try {
+      beneIds.add(Long.parseLong(beneId));
+    } catch (NumberFormatException e) {
+      LOGGER.warn("Could not parse long from bene_id: " + beneId);
```

Comment:
I don't think we ended up needing this here after the refactor

Reply:
removed